src/lib/menu/menu-directive.ts

```diff
@@ -52,6 +52,7 @@ export class MdMenu implements AfterContentInit, MdMenuPanel, OnDestroy {
 
   @ViewChild(TemplateRef) templateRef: TemplateRef<any>;
   @ContentChildren(MdMenuItem) items: QueryList<MdMenuItem>;
+  @Input('overlap-trigger') overlapTrigger = true;
```

Comment:
Don't need to rename to dash-case, we're trying to switch everything to camelCase